File: Java/LogicaProgramacaoJava(Eclipse)/exerciciosJava/src/manzanoPag25e26/ExercicioH.java
package manzanoPag25e26;

import java.util.Scanner;

/*h) Elaborar um programa que calcule e apresente o volume de uma caixa retangular, por meio da fórmula
VOLUME <- COMPRIMENTO * LARGURA * ALTURA */

public class ExercicioH {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		int volume, comprimento, largura, altura;

		System.out.println("Informe o comprimento da caixa:");
		comprimento = sc.nextInt();
		System.out.println("Informe a largura da caixa:");
		largura = sc.nextInt();
		System.out.println("Informe a altura da caixa:");
		altura = sc.nextInt();

		volume = comprimento * largura * altura;
		System.out.println("O volume da caixa é: " + volume);

		sc.close();

	}

}

File: LogicaDeProgramacao(JavaScript)/exercicios pag 25 e 26(Manzano)/js/exercicioH.js
/*h) Elaborar um programa que calcule e apresente o volume de uma caixa retangular, por meio da fórmula
VOLUME <- COMPRIMENTO * LARGURA * ALTURA */


var volume = 0


var comprimento = parseInt(prompt("Informe o comprimento da caixa:"))
var largura = parseInt(prompt("Informe a largura da caixa:"))
var altura = parseInt(prompt("Informe a altura da caixa:"))
volume = comprimento*largura*altura
alert("O volume da caixa é: "+volume)